scripts/temp.deployment.sh

```diff
@@ -21,24 +22,50 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
-# Temporary script to deploy argo onto a system for testing
-#     NOTE: this will be removed once we have deployment work done
+set -e
 
-# csm 1.2
+function deployNLS() {
+    BUILDDIR="/tmp/build"
+    mkdir -p "$BUILDDIR"
+    kubectl get secrets -n loftsman site-init -o jsonpath='{.data.customizations\.yaml}' | base64 -d > "${BUILDDIR}/customizations.yaml"
+    kubectl get configmap -n loftsman loftsman-platform -o jsonpath='{.data.manifest\.yaml}' > "${BUILDDIR}/iuf.yaml"
+    manifestgen -i "${BUILDDIR}/iuf.yaml" -c "${BUILDDIR}/customizations.yaml" -o "${BUILDDIR}/platform.yaml"
+    yq w -i "${BUILDDIR}/platform.yaml" 'spec.charts[0].version' "$2"
+    charts="$(yq r /tmp/build/platform.yaml 'spec.charts[*].name')"
+    for chart in $charts; do
+        if [[ $chart != "cray-iuf" ]] && [[ $chart != "cray-nls" ]]; then
+            yq d -i /tmp/build/platform.yaml "spec.charts.(name==$chart)"
+        fi
+    done
+
+    yq d -i /tmp/build/platform.yaml "spec.sources"
+
+    loftsman ship --charts-path "$1" --manifest-path /tmp/build/platform.yaml
```

Comment:
It looks like this overwrites the value of the configmap `loftsman-platform` in the `loftsman` manifest. This is probably pretty harmless, but IMO, it's a bad practice as it changes something that was put in place by the normal CSM install on a system.

For comparison, right now on frigg:

```
ncn-m001:/mnt/developer/haasken # kubectl get configmap -n loftsman loftsman-platform -o jsonpath='{.data.manifest\.yaml}' | yq r - 'spec.charts[*].name' | wc -l
2
```

vs. on shandy where I assume this script has not been (and should not be) run:

```
ncn-m001:~ # kubectl get configmap -n loftsman loftsman-platform -o jsonpath='{.data.manifest\.yaml}' | yq r - 'spec.charts[*].name' | wc -l
45
```

Perhaps we can use a different file name with this `loftsman ship` to avoid this.